.github/workflows/testAndPublish.yml

```diff
@@ -0,0 +1,314 @@
+name: Run tests and publish
+
+on:
+  push:
+    branches:
+    - master
+    - beta
+    - rc
+    - 'try-**'
+    tags:
+    - 'release-**'
+
+  pull_request:
+    branches:
+    - master
+    - beta
+    - rc
+
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  START_BUILD_NUMBER: 50000
+  pullRequestNumber: ${{ github.event_name == 'pull_request' && github.event.number || 0 }}
+  scons_publisher: NV Access
```

Comment:
Why is this set here explicitly?

Reply:
so forks can easily change it. It might be even better as a github env var

Reply:
All the other feature variables are placed here for the same reason